terraform/create_backend_vars.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
The purpose of this script is to easily generate a valid backend_config.tfvars file for local Terraform deployments that need or alter the remote state.